dhctl/pkg/operations/destroy/destroy.go

```diff
@@ -112,7 +114,38 @@ func NewClusterDestroyer(params *Params) (*ClusterDestroyer, error) {
 
 	clusterInfra := infra.NewClusterInfraWithOptions(terraStateLoader, state.cache, params.TerraformContext, infra.ClusterInfraOptions{PhasedExecutionContext: pec})
 
-	staticDestroyer := NewStaticMastersDestroyer(wrapper.Client())
+	ctx, cancel := context.WithTimeout(context.Background(), 30*time.Second)
```

Comment:
first, please use retry package for getting nodes ip's
second, let to do standalone function for this routine and let move getting nodes ip's to https://github.com/deckhouse/deckhouse/blob/main/dhctl/pkg/operations/destroy/destroy.go#L166

Reply:
Done